example/examplechecker.py

```diff
@@ -160,12 +159,13 @@ def getnoise(self) -> None:
         """
         This method retrieves noise in the service.
         The noise to be retrieved is inside self.flag
```

Comment:
Docstring outdated
self.flag is None
(I believe i updated it in the unique_identifier branch)